test/handler.go

```diff
@@ -40,10 +40,12 @@ func NewHandler() *Handler {
 	h := &Handler{
 		Handler: pilosa.NewHandler(),
 	}
-	h.Handler.Executor = &h.Executor
+	h.API = pilosa.NewAPI()
+	h.Handler.API = pilosa.NewAPI()
```

Comment:
should this be `h.Handler.API = h.API` instead? perhaps it doesn't matter, but I'm wondering if having those separate might cause confusion in the future.

Reply:
I think you're right